### DISCUSSION TITLE
General discussion

### DISCUSSION THREAD

**Discussion by ehristoforu** [ehristoforu/DeFooocus#6] [General]
Here we talk about general topics

**Comment by metatron182** [comment]
Hi, Can someone please help me with the installation of DeFooocus. When I run the install file, it Disappears. and nothing happens. when I run the run file, it gives me this message.

\python_embeded\python.exe: can't open file 'E:\\Defooocus\\DeFooocus_win64_portable\\DeFooocus\\entry_with_update.py': [Errno 2] No such file or directory
Press any key to continue . . .

https://github.com/ehristoforu/DeFooocus/assets/167347969/7b53cc9b-6612-4faa-99c3-9114ab9ca558

**Reply by werzieq1** [reply]
Same problem xD

**Comment by metatron182** [comment]
DeFooocus package needs to be available in Stability Matric? It would be a lot easier to installed.

**Comment by eeeee112** [comment]
Hello, will you add Fooocus-API as in Fooocus-API answer please.
